Detectors/TRD/simulation/src/TrapSimulator.cxx

```diff
@@ -722,15 +741,23 @@ void TrapSimulator::draw(int choice, int index)
       trklLines[iTrkl].SetLineColor(2);
       trklLines[iTrkl].SetLineWidth(2);
       LOG(info) << "Tracklet " << iTrkl << ": y = " << trkl.getY() << ", dy = " << (trkl.getdY() * 140e-4) << " offset : " << offset;
-      //TODO put that 140e-4 in some constant somewhere
+      LOG(info) << "Tracklet " << iTrkl << ": x1,y1,x2,y2 :: " << trklLines[iTrkl].GetX1() << "," << trklLines[iTrkl].GetY1() << "," << trklLines[iTrkl].GetX2() << "," << trklLines[iTrkl].GetY2();
+      LOG(info) << "Tracklet " << iTrkl << ": t0 : " << t0 << ", t1 " << t1 << ", padwidth:" << padWidth << ", slope:" << slope;
 
       trklLines[iTrkl].Draw();
     }
   }
+  //hist->Write();
+  // histfiltered->Write();
+  // for (int iTrkl = 0; iTrkl < mTrackletArray.size(); iTrkl++) trklLines[iTrkl].Write();
+  // grHits->Write();
+  //  hist->release();
+  //  grHits->release();
```

Comment:
do you need these new comments (which seem to be code)?

Reply:
gone